packages/vega-parser/src/parsers/guides/guide-util.js

```diff
@@ -86,3 +87,80 @@ export function extendOffset(value, offset) {
     : !isObject(value) ? { value, offset }
     : { ...value, offset: extendOffset(value.offset, offset) };
 }
+
+const DISCRETE_SCALES = new Set(['ordinal', 'band', 'point']);
+
+function formatType(_, scaleType) {
+  const formatType = _('formatType');
+
+  if (formatType) {
+    return formatType;
+  }
+  if (scaleType === Time) {
+    return 'time'
+  }
+  if (scaleType === UTC) {
+    return 'utc'
+  }
+}
+
+function wrapFormat(text, format, formatType) {
+  if (format == null) {
+    return text
+  }
+  const func = formatType === "time" ? "timeFormat" : formatType === "utc" ? "utcFormat" : "format";
```

Comment:
Should use single quotes

Reply:
Done. Enforced in https://github.com/vega/vega/pull/2514.